src/SFA.DAS.RequestApprenticeTraining.Database/Tables/Standard.sql

```diff
@@ -1,8 +1,8 @@
 ﻿CREATE TABLE [dbo].[Standard]
 (
-    [StandardReference] VARCHAR(25) NOT NULL PRIMARY KEY, 
-    [StandardTitle] VARCHAR(100) NOT NULL,
+    [StandardReference] NVARCHAR(6) NOT NULL PRIMARY KEY, 
```

Comment:
The change from 25 to 6 is understandable but why is a 'n' varchar needed - what not ascii character would be expected here? - or has that just been used in the Courses API? so you are following that through?

Reply:
Changed to nvarchar(6) to match EmployerRequest.StandardReference, but its just a varchar in the Courses DB.  Should I change them both to varchar(6)

Reply:
Changed Standard.StandardReference and EmployerRequest.StandardReference to varchar(6)